benchmarks/span-labeling-datasets/project.yml

```diff
@@ -0,0 +1,501 @@
+title: "Spancat datasets"
+description: |
+  This project compiles various NER and more general spancat datasets 
+  and their converters into the [spaCy format](https://spacy.io/api/data-formats). 
+  You can use this to try out experiment with `ner` and `spancat`
+  or to potentially pre-train them for your application.
+
+vars:
+  spans_key: "sc"
+  gpu_id: 0
+
+directories:
+  - "assets"
+  - "configs"
+  - "corpus/spancat"
+  - "corpus/ner"
+  - "scripts"
+  - "analyses"
+  - "unseen"
+
+workflows:
+  wnut17:
+    - "preprocess-wnut17"
+    - "convert-wnut17-ents"
+    - "convert-wnut17-spans"
+    - "inspect-wnut17"
+  conll:
+    - "unpack-conll"
+    - "preprocess-conll"
+    - "convert-conll-ents"
+    - "convert-conll-spans"
+    - "inspect-conll"
+  archaeo:
+    - "convert-archaeo-ents"
+    - "convert-archaeo-spans"
+    - "clean-archaeo"
+    - "inspect-archaeo"
+  anem:
+    - "convert-anem-ents"
+    - "convert-anem-spans"
+    - "inspect-anem"
+  restaurant:
+    - "preprocess-restaurant"
+    - "convert-restaurant-ents"
+    - "convert-restaurant-spans"
+    - "inspect-restaurant"
+  all:
+    - "preprocess-wnut17"
+    - "convert-wnut17-ents"
+    - "convert-wnut17-spans"
+    - "unpack-conll"
+    - "convert-conll-spans"
+    - "convert-conll-ents"
+    - "convert-archaeo-ents"
+    - "convert-archaeo-spans"
+    - "convert-anem-ents"
+    - "convert-anem-spans"
+    - "preprocess-restaurant"
+    - "convert-restaurant-ents"
+    - "convert-restaurant-spans"
+    - "inspect-restaurant"
+
+assets:
+  # WNUT17 https://aclanthology.org/W17-4418/
```

Comment:
I think it would be nice to put these URLs in the `description` field per-asset, that way they get included in the description in the README when you do `spacy project document`, and they aren't hidden in the `project.yml` file.